inlang/source-code/paraglide/paraglide-js/src/compiler/compile.test.ts

```diff
@@ -221,6 +221,33 @@ describe("e2e", async () => {
 			runtime.onSetLanguageTag(() => {})
 		}).toThrow()
 	})
+
+	test("should return the correct message if a languageTag is set in the message options", async () => {
+		const { m, runtime } = await import(
+			`data:application/javascript;base64,${Buffer.from(
+				compiledBundle.output[0].code,
+				"utf8"
+			).toString("base64")}`
+		)
+
+		// set the language tag to de to make sure that the message options override the runtime language tag
+		runtime.setLanguageTag("de")
```

Comment:
Change the test to (more is not needed)

```ts
runtime.setLanguageTag("de")
expect(m.onlyText()).toBe("Eine einfache Nachricht.")
expect(m.onlyText(undefined, { languageTag: "en" })).toBe("A simple message.")
```

- ensure that the same message still works without the language tag option
- verifies the language tag option